public/static/locales/en/tools.json

```
@@ -8,6 +8,7 @@
     "attribution": "Attribution",
     "bindToHost": "Bind to Host",
     "cancel": "Cancel",
+    "confirmDelete": "Do you really want to delete this tool?",
```

Should this say "Delete the selected tools", since the user can delete more than 1 at once?
```suggestion
    "confirmDelete": "Delete the selected tools?",
```